### PR TITLE
Document non-public cloud support for Azure Monitor scaler.

### PR DESCRIPTION
**Pull request by v-shenoy**
Signed-off-by: Vighnesh Shenoy <vshenoy@microsoft.com>

Update Azure monitor scaler documentation.
https://github.com/kedacore/keda/issues/1917

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.7/scalers/azure-monitor.md [hunk 1]
```diff
@@ -44,6 +50,9 @@ triggers:
 - `metricAggregationInterval` - Collection time of the metric in format `"hh:mm:ss"` (Default: `"0:5:0"`, Optional)
 - `activeDirectoryClientId` - Id of the Active Directory application which requires at least `Monitoring Reader` permissions. (Optional)
 - `activeDirectoryClientPasswordFromEnv` - Name of the environment variable that contains the active directory client password. (Optional)
+- `cloud` - Name of the cloud environment that the Azure resource belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureGermanCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
+- `resourceManagerEndpoint` - Resource manager endpoint of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://management.chinacloudapi.cn/` for `AzureChinaCloud`).
```
Comment (tomkerkhove) [inline content/docs/2.7/scalers/azure-monitor.md:54]: Maybe we should prefix it with Azure given it's referred to as ARM?
```suggestion
- `azureResourceManagerEndpoint` - Azure Resource manager endpoint of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://management.chinacloudapi.cn/` for `AzureChinaCloud`).
```

**Review by tomkerkhove — APPROVED** [review]
LGTM

File: content/docs/2.7/scalers/azure-monitor.md (hunk 1 above)
Comment (v-shenoy) [reply content/docs/2.7/scalers/azure-monitor.md]: Done.

**Review by JorTurFer — APPROVED** [review]
LGTM!

**Pull request merged**